pages/en/cookbook/migrating-a-subgraph.mdx

```diff
@@ -63,17 +65,68 @@ graph codegen && graph build
+
+![API key creation page](/img/api-image.png)
+
+At the end of each week, an invoice will be generated based on the query fees that have been incurred during this period. This invoice will be paid automatically using the GRT available in your balance. Your balance will be updated after the cost of your query fees are withdrawn. Query fees are paid in GRT via the Polygon network. You will need to add GRT to the Polygon billing contract to enable your API key via the following steps:
+
+- Purchase GRT on an exchange of your choice
+- Send the GRT to your wallet
+- On the Billing page in Studio, click on Add GRT
+
+![Add GRT in billing](/img/add-grt-in-billing.png)
+
+- Follow the steps to bridge the GRT to Polygon.
+- Once bridged, add the GRT to your billing balance
+
+![Billing pane](/img/billing-pane.png)
+
+> Note: This process is slightly different if you are using a multisig since you can not bridge GRT to Polygon with a multisig. You would need to send the GRT to an EOA (externally owned address) and bridge it from there to Polygon. Once on Polygon, you can load the multisig address. We have created a special tool to help you through these steps: https://multisig-billing.thegraph.com/
+
+It is recommended that you secure the API by limiting its usage in two ways:
+
+1. Authorized Subgraphs
+2. Authorized Domain
+
+![Subgraph lockdown page](/img/subgraph-lockdown.png)
+
+### Querying your subgraph on the decentralized network
+
+Now you can check the indexing status of the Indexers on the network in Graph Explorer (example [here](https://thegraph.com/explorer/subgraph?id=S9ihna8D733WTEShJ1KctSTCvY1VJ7gdVwhUujq4Ejo&view=Indexers)). The green line at the top indicates that at the time of posting 8 Indexers successfully indexed that subgraph. Also in the Indexer tab you can see which Indexers picked up your subgraph.
+
+![Rocket Pool subgraph](/img/rocket-pool-subgraph.png)
+
+As soon as the first Indexer has fully indexed your subgraph you can start to query the subgraph on the decentralized network. In order to retrieve the query URL for your subgraph, you can copy/paste it by clicking on the symbol next to the query URL. You will see something like this:
+
+`https://gateway.thegraph.com/api/[api-key]/subgraphs/id/S9ihna8D733WTEShJ1KctSTCvY1VJ7gdVwhUujq4Ejo`
+
+Important: Make sure to replace `[api-key]` with an actual API key generated in the section above.
+
+You can now use that Query URL in your dapp to send your GraphQL requests to.
+
+Congratulations! You are now a pioneer of decentralization!
+
+> Note: Due to the distributed nature of the network it might be the case that different Indexers have indexed up to different blocks. In order to only receive fresh data you can specify the minimum block an Indexer has to have indexed in order to serve your query with the block: `{ number_gte: $minBlock }` field argument as shown in the example below:
+
+`{ stakers(block: { number_gte: 14486109 }) { id } }`
```

Comment:
Should this be a code block instead?

````suggestion
```
{
  stakers(block: { number_gte: 14486109 }) {
    id
  }
}
```
````

Reply:
Yes, I think a code block here works